test/datasets_utils.py

```diff
@@ -581,6 +581,28 @@ def test_transforms(self, config):
 
                 mock.assert_called()
 
+    @test_all_configs
+    def test_transforms_v2_wrapper(self, config):
+        # This is stable test, so we can't depend on prototype stuff
+        try:
+            from torchvision.prototype.datapoints import wrap_dataset_for_transforms_v2
+        except ImportError:
+            return
+
+        try:
+            with self.create_dataset(config) as (dataset, _):
+                wrapped_dataset = wrap_dataset_for_transforms_v2(dataset)
+                wrapped_sample = wrapped_dataset[0]
+                assert wrapped_sample is not None
```

Comment:
This currently fails for `DatasetFolder`, since it doesn't load anything:

https://github.com/pytorch/vision/blob/87ec8048085a5d892e78e60cd1fb0b4e37219c1e/test/test_datasets.py#L1551

But, I'll have to tackle that test anyway for another reason: https://github.com/pytorch/vision/pull/7155#issuecomment-1423267645. I think I can manage something that is sufficient for both cases.

Reply:
Fixed in #7216.